lua/gitlab/init.lua

```diff
@@ -74,9 +74,8 @@ return {
     draft_notes_dep,
     discussion_data,
   }, discussions.toggle),
-  toggle_resolved = async.sequence({ info }, discussions.toggle_discussion_resolved),
+  toggle_draft_mode = async.sequence({ info }, discussions.toggle_draft_mode),
   publish_all_drafts = draft_notes.publish_all_drafts,
-  reply = async.sequence({ info }, discussions.reply),
```

Comment:
Don't think this is quite right, you don't want to be getting rid of these other mappings. You also don't need to wrap this in `async.sequence` which is used to fetch data via the Gitlab API.

Reply:
I've removed the unnecessary async call. But I still don't see how `discussions.toggle_discussion_resolved` and `discussions.reply`, or `gitlab.toggle_resolved` and `gitlab.reply`, respectively, could be used outside of the plugin. These functions are not even described in the API documentation. How would a user supply the necessary `tree` argument? So I thought they were included in `lua/gitlab/init.lua` by mistake. If you want, I can get them back, of course.

Reply:
oh yeah you're right, good call